### DISCUSSION TITLE
Donation Currency Exchange

### DISCUSSION THREAD

**Discussion by TokynBlast** [vim/website_next_generation#5] [Ideas]
You should get the location, then based off of where they live, like in the us, for donations, it will say '10.74 USD', you would use stock market info to exchange between them, so that way you always get 10 euros out of it. In Europe, it would just always be 10 euros, because that is the base currency

**Comment by manuel-pross** [comment]
This is a really detailed question. We have not thought about that yet.
At the moment and in the future all donations are done via Paypal.

**Reply by sidarth-23** [reply]
Currently my country India and many more do not support PayPal transactions. So maybe we should implement some means for wired transaction like visa and MasterCard transfers

**Reply by chrisbra** [reply]
Guys, this is something for a banking website and way too copmlex to do right for a open source project. Perhaps one can do a popup showing you what the current amount is in any other (selected) currency. But this can always be only an estimate. Also regarding paypal or not paypal, this needs to be driven by the ICCF not by us here.
